### ISSUE TITLE
Screen reader does not announce the name and role of menu items in scan mode

### ISSUE THREAD

**Issue by RajeshGoriga** [microsoft/fluentui#19162]
### Environment Information

- **Package version(s)**: v8

### Describe the issue:

Screen reader not announcing name and role of menu items in Scan mode  

#### Actual behavior:

Screen reader not announcing name and role of menu items in Scan mode  

#### Expected behavior:

It Should read out name and role of menu items in scan mode 

### Documentation describing expected behavior

It not not reading as expected

https://user-images.githubusercontent.com/13463251/127292732-f3299852-46b0-4be7-a477-78b821910a08.mp4

**Comment by ling1726** [comment]
@RajeshGoriga I am using Narrator on the docsite examples: https://developer.microsoft.com/en-us/fluentui#/controls/web/contextualmenu and `menuitem` is narrated. 

Can you provide some more information such as what screen reader you are using and a code example similar to what you have in your app ?

**Comment by RajeshGoriga** [comment]
When i use NVDA + Chrome browser in office admin portal it is not reading and Narrator + EDGE is working as expected  
![image](https://user-images.githubusercontent.com/13463251/127982310-009b20ba-aaa4-4931-9e80-7b1dd1b1874e.png)
And i have pass items like this

**Comment by ling1726** [comment]
@RajeshGoriga please provide a code example of your issue, since it is hard for us to determine whether this is an issue in Fluent or the application code used with it in your case
